src/posit/__init__.py

```diff
@@ -1 +1,3 @@
 """The Posit SDK."""
+
+from .connect import Connect  # noqa
```

Comment:
Is it confusing that both `from posit import connect` and `from posit import Connect` are correct but do very different things? 

Even if it is a good idea (not making a judgment on that), it will probably cause hell for generating documentation on case-insensitive file systems (macOS) because we'll have man pages for `posit.connect` and `posit.Connect` colliding with each other. On Arrow for example, I could basically never touch the sphinx docs because of collisions like this.

Reply:
That's an interesting edge case. I haven't run into it before, but it makes perfect sense.

After doing some more digging, I noticed that the pattern we currently have seems common (Google, AWS). The only difference I see is that in SDKs where `Client` is used without a service name prefix, the reference documentation typically follows this pattern:

(aside; it's also common in other languages that promote namespacing via packages. Java and Go, for example.)

```py
from posit import connect

client = connect.Client()
```

The other common approach (Stripe, Mongo, Azure) is prefixing the product name (i.e., `ConnectClient`).

```py
from posit import ConnectClient

client = ConnectClient()
```

I personally like the first/current option more since it provides a generic namespace for any other "Connect" specific code. However, the second option is also quite nice due to its simplicity and verbosity. If I had to bet, I would assume the popular vote would be for `ConnectClient`. Therefore, I think our best option is to do both via:

`src/posit/__init__.py`
```py
from posit.connect import Client as ConnectClient
```

Reply:
This will extend nicely to other products as well.

```py
from posit.connect import Client as ConnectClient
from posit.package_manager import Client as PackageManagerClient
from posit.workbench import Client as WorkbenchClient
```

Reply:
My preference would be to have `one-- and preferably only one --obvious way to do it`.
```
from posit.connect import Client
client = Client()
```

seems idiomatic to me, as does

```
from posit import connect
client = connect.Client()
```

I would probably only use the latter if I was planning on talking to multiple Posit products in the same script, or if I needed something else from the `connect` namespace.